iocore/net/UnixUDPNet.cc

```diff
@@ -923,7 +923,7 @@ UDPNetHandler::startNetEvent(int event, Event *e)
   (void)event;
   SET_HANDLER((UDPNetContHandler)&UDPNetHandler::mainNetEvent);
   trigger_event = e;
-  e->schedule_every(-HRTIME_MSECONDS(UDP_PERIOD));
+  e->schedule_every(-HRTIME_MSECONDS(UDP_PERIOD) - 1);
```

Comment:
Make sure PollCont(run in ET_UDP) always run before UDPNetHandler in the same loop.

Reply:
It might be better to define constants. It would be failed if someone changed the value for PollCont.

Reply:
will fixed !